src/components/ConfirmedActionButton/__tests__/ConfirmedActionButton.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { ConfirmedActionButton } from '../ConfirmedActionButton';
```

Comment:
Absolute import again

Reply:
we use relative import in tests (unless we are importing a component that is not the primary component being tested)

Reply:
Cool. Could you please update the code style guide to mention that